nautobot/core/management/commands/generate_performance_test_endpoints.py

```diff
@@ -0,0 +1,57 @@
+import collections
+
+from django.core.management.base import BaseCommand
+from django.urls import get_resolver
+
+
+class Command(BaseCommand):
+    """
+    Example usage: `nautobot-server generate_performance_test_endpoints > endpoints.yml`
+    """
+
+    help = "List all relevant performance test url patterns in Nautobot Core"
+
+    def handle(self, *args, **options):
+        # Get the URL resolver
+        url_patterns = get_resolver().url_patterns
+
+        # Group the urls by app names
+        self.app_name_to_urls = collections.defaultdict(list)
+        # Fetch and store the urls by app names in the dictionary
+        self.fetch_urls(url_patterns)
+
+        # Print the urls out in .yml format
+        self.stdout.write("---")
+        self.stdout.write("endpoints:")
+        for _, urls in self.app_name_to_urls.items():
+            for url in urls:
+                pattern_name, url_pattern = url  # ["dcim:location_list:", "/dcim/locations/"]
+                self.stdout.write(f"  {pattern_name}")  # "dcim:location_list:"
+                self.stdout.write(f"    - {url_pattern}")  # "/dcim/locations/"
+
+    def fetch_urls(self, url_patterns):
+        for pattern in url_patterns:
+            if hasattr(pattern, "url_patterns"):
+                # If it's a nested URL pattern, recursively list its URLs
+                self.fetch_urls(pattern.url_patterns)
+            else:
+                # Only fetch urls from relevant apps
```

Comment:
We'll almost certainly want at some point to be able to include Nautobot App endpoints in addition to those from core...